webapp/graphite/account/ldapBackend.py

```diff
@@ -12,7 +12,9 @@
 See the License for the specific language governing permissions and
 limitations under the License."""
 
-import ldap, traceback
+import traceback
+
```

Comment:
Just curious why the blank line here?

Reply:
It is a somewhat less consistent convention, to put standard library imports first, then a blank line, then third-party imports, then a blank line, then project-internal imports.

Reply:
Ok.  Thanks for the explanation.